### PR TITLE
chore: add coffee warehouse nextjs app

### PR DESCRIPTION
**Pull request by WissamProgress**
Related to https://github.com/telerik/kendo-react-private/issues/3259

### CODE REVIEW

File: examples/coffee-warehouse-nextjs/LICENSE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+This project has been released under the [Apache License, version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html), the text of which is included below. This license applies ONLY to the source of this repository and does not extend to any other Kendo UI distribution or variant, or any other 3rd party libraries used in a repository. For licensing information about KendoReact, see the [License Agreements page](https://www.telerik.com/purchase/license-agreement/progress-kendoreact) at [KendoReact.com](http://www.kendoreact.com).
+
+> Copyright © 2020 Progress Software Corporation and/or its subsidiaries or affiliates.
```
Comment (elena-gancheva) [inline examples/coffee-warehouse-nextjs/LICENSE:3]: we should change the year

File: examples/coffee-warehouse-nextjs/app/drawer/profile/page.jsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
+'use client';
+import * as React from 'react';
+
+import { useLocalization } from '@progress/kendo-react-intl';
+//import { useNavigate } from "react-router-dom";
```
Comment (elena-gancheva) [inline examples/coffee-warehouse-nextjs/app/drawer/profile/page.jsx:5]: make sure we do not show commented code

File: examples/coffee-warehouse-nextjs/app/drawer/profile/page.jsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+import { Input } from '../../components/form/Input';
+import { MaskedTextBox } from '../../components/form/MaskedTextBox';
+import { DropDownList } from '../../components/form/DropDownList';
+import { Editor } from '../../components/form/Editor';
+import { Upload } from '../../components/form/Upload';
+import { RadioGroup } from '../../components/form/RadioGroup';
+import { Switch } from '../../components/form/Switch';
+
+//import { AppContext } from './../AppContext'
```
Comment (elena-gancheva) [inline examples/coffee-warehouse-nextjs/app/drawer/profile/page.jsx:17]: same here

File: examples/coffee-warehouse-nextjs/app/drawer/profile/page.jsx [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+const countriesData = countries.map(country => country.name);
+const teamsData = teams.map(team => ({
+    value: team.teamID,
+    label: team.teamName
+}));
+
+const noMessage = 'message not defined';
+
+export default function Profile() {
+    //const {languageId, onLanguageChange, onProfileChange, ...formValues} = React.useContext(AppContext);
```
Comment (elena-gancheva) [inline examples/coffee-warehouse-nextjs/app/drawer/profile/page.jsx:33]: and here

**Comment by kspeyanski** [comment]
Please let me know once this is merged. I will proceed with deployment outside of github!

**Review by elena-gancheva — APPROVED** [review]

**Pull request merged**